### ISSUE TITLE
Copy-pasting text with an inline void from Slate to other editors is not working as expected. 

### ISSUE THREAD

**Issue by laufeyrut** [ianstormtaylor/slate#4802]
**Description**
Copy-pasting text with an inline void from Slate to other editors is not working as expected. 

**Recording**
https://user-images.githubusercontent.com/15036127/150869415-33422c13-2666-4e58-9104-2464474b7107.mov

**Steps**
To reproduce the behavior:
1. Go to https://www.slatejs.org/examples/mentions
2. Copy the text from the example 
3. Paste the text to some editor. Like Google Docs or notes. 
4. There is a new line after each inline element

<img width="406" alt="Screenshot 2022-01-24 at 20 53 56" src="https://user-images.githubusercontent.com/15036127/150865811-2269779b-80d9-4709-8044-0b6f82f6e943.png">
<img width="476" alt="Screenshot 2022-01-24 at 20 53 30" src="https://user-images.githubusercontent.com/15036127/150865816-6a5df56b-ab77-4cc2-bddc-f8f14704a1eb.png">

**Expectation**
That and new line is not added after inline elements
<img width="523" alt="Screenshot 2022-01-24 at 21 10 57" src="https://user-images.githubusercontent.com/15036127/150865890-d186c99c-f720-467c-9d33-63f486136e24.png">

**Environment**
- Slate Version: [0.72.3
- Operating System: MacOs 11.6.2
- Browser: Microsoft Edge

**Comment by jackmiller2708** [comment]
@laufeyrut I agree. As of right now, Slate is using the zero width no-break space (U+FEFF), which is deprecated. They should switch up the default to word joiner (U+2060) to represent void characters instead.

**Comment by yume-chan** [comment]
Using `U+2060` won't fix the issue in Google Docs. It's caused by this "position: absolute" style:

https://github.com/ianstormtaylor/slate/blob/55b9574097f6008bda7ed8e3cb7aa9dd607d9f49/packages/slate-react/src/components/element.tsx#L111-L127

**Comment by yume-chan** [comment]
Use `U+2060`: #2597, use `<br>`: #1971
